Makefile

```diff
@@ -1,18 +1,33 @@
-.PHONY: dev clean e2e push
+.PHONY: clean dev-public-cluster update-image-on-deployment push-tester-image deploy-e2e run-e2e
 
-dev:
-	terraform --version
-	cd devenv && mkdir -p state && terraform init && terraform apply -auto-approve
 
-clean:
-	rm -rf devenv/state devenv/.terraform.lock.hcl devenv/.terraform devenv/terraform.tfstate devenv/terraform.tfstate.backup
+# keep separate for simultaneous public/private dev without need for resource recreation
+clean-public:
+	rm -rf devenv/state devenv/public_cluster_tf/.terraform.lock.hcl devenv/public_cluster_tf/.terraform devenv/public_cluster_tf/terraform.tfstate devenv/public_cluster_tf/terraform.tfstate.backup
 
-e2e:
-	go test -v --count=1 --tags=e2e ./e2e
+dev-public-cluster:
```

Comment:
this should just be `dev-public`. Would match `clean-public` and shorter is better because we are running these commands all the time.

Reply:
makes sense